docs/content/docs/faq.md

```diff
@@ -62,6 +62,24 @@ one Fluid service.
 Fluid clients connect to the Fluid service using the WebSocket protocol. However, the Fluid runtime manages
 all of the connections so that Fluid client developers can focus on local experiences.
 
+### What are data stores used for?
+
+Data stores are used to collect DDSes under a single parent. A DDS must be parented by to a data store and you can treat data stores to be collections of DDSes.
+
+### What is the difference between data stores and data objects?
+
+In core FluidFramework we have data stores. As the name suggests, it is a place where you can store data. It contains one or more DDSs that have the actual user data. A data store is responsible (among other things) for serializing this data to be stored (in a file) and then re-hydrating this data when the file is loaded.
```

Comment:
@tylerbutler, isn't "DDSs" supposed to cause a build break?

Reply:
It should! Will file an issue.

Reply:
#9978